tests/behat/behat-features/src/ContentPoolContext.php

```diff
@@ -203,4 +231,32 @@ public function seeCurrentSiteRegistered() {
     }
   }
 
+  /**
+   * @Given I click on media :media_title in entity browser :entity_browser
+   */
+  public function iClickOnMediaInEntityInBrowser($media_title, $entity_browser) {
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\".views-field-name:contains('$media_title')\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\".views-field-name:contains('$media_title')\").first().closest(\".views-row\").click()");
+  }
+
+  /**
+   * @Then Value of input field :element_selector is :value
+   */
+  public function inputHasValue($element_selector, $value) {
+
+    $el = $this->getSession()->getPage()->find('css', $element_selector);
```

Comment:
always use full variable names, no unclear abbreviations - element.

Reply:
This should move into behat-drupal-utils and work with general selectors (xpath) like others, see MinkElementTrait.
However, is it really not available? there are steps for check for unchecked checkboxes, are the not working? Look them up with

source dotenv/loader.sh; docker-compose exec web ./vendor/bin/behat -c web/modules/drunomics/contentpool-client/tests/behat/behat.yml -d SEARCHKEYWORD

Reply:
It is available for checkboxes. But this is not a checkbox. This is a hidden input field (but this also covers other types too) that contains referenced entity id.